devstack/plugin.sh

```diff
@@ -81,8 +79,8 @@ if is_service_enabled group-policy; then
         if [[ $ENABLE_NFP = True ]]; then
             echo_summary "Configuring $NFP"
             nfp_configure_neutron
-            [[ $DEVSTACK_MODE = enterprise ]] && nfp_configure_nova
-            if [[ $DEVSTACK_MODE != base ]]; then
+            [[ $NFP_DEVSTACK_MODE = enterprise ]] && nfp_configure_nova
```

Comment:
Lets make the checks as base or advanced mode. Enterprise code changes and checks should be pluggable on top of submitted code.